### PR TITLE
Changing cursor to pointer upon hover

### PR DESCRIPTION
**Pull request by zeeshanibnali**
So whenever you hover over the header navigation on top right, which includes the notifications icon, user profile drop-down, etc, the cursor turns into a **pointer**.

And also added a log to know that the database has been connected successfully.

### CODE REVIEW

**Comment by zeeshanibnali** [comment]
This pull request is for the issue:
[https://github.com/idurar/idurar-erp-crm/issues/189](url)

File: frontend/src/app/HeaderContent/index.jsx [hunk 1]
```diff
@@ -76,13 +76,13 @@ export default function HeaderContent() {
     <div className="headerIcon" style={{ position: 'absolute', right: 0, zIndex: '99' }}>
       <Dropdown overlay={profileDropdown} trigger={['click']} placement="bottomRight">
         {/* <Badge dot> */}
-        <Avatar className="last" src={photo} />
+        <Avatar className="last cursor" src={photo} />
```
Comment (belyas) [inline frontend/src/app/HeaderContent/index.jsx:79]:
```suggestion
        <Avatar className="last cursor-pointer" src={photo} />
```

File: frontend/src/app/HeaderContent/index.jsx [hunk 2]
```diff
@@ -76,13 +76,13 @@ export default function HeaderContent() {
...
         {/* </Badge> */}
       </Dropdown>
 
-      <Avatar icon={<AppstoreOutlined />} />
+      <Avatar className="cursor" icon={<AppstoreOutlined />} />
```
Comment (belyas) [inline frontend/src/app/HeaderContent/index.jsx:83]:
```suggestion
      <Avatar className="cursor-pointer" icon={<AppstoreOutlined />} />
```

File: frontend/src/app/HeaderContent/index.jsx [hunk 3]
```diff
@@ -76,13 +76,13 @@ export default function HeaderContent() {
...
 
-      <Avatar icon={<BellOutlined />} />
+      <Avatar className="cursor" icon={<BellOutlined />} />
```
Comment (belyas) [inline frontend/src/app/HeaderContent/index.jsx:85]:
```suggestion
      <Avatar className="cursor-pointer" icon={<BellOutlined />} />
```

File: frontend/src/style/partials/header.less [hunk 4]
```diff
@@ -14,6 +14,10 @@
   .last {
     margin-right: 30px;
   }
+
+  .cursor {
```
Comment (belyas) [inline frontend/src/style/partials/header.less:18]:
```suggestion
  .cursor-pointer {
```

File: server.js [hunk 5]
```diff
@@ -14,7 +14,10 @@ require('dotenv').config({ path: '.variables.env' });
 // Connect to our Database and handle any bad connections
 // mongoose.connect(process.env.DATABASE);
 
-mongoose.connect(process.env.DATABASE);
+mongoose
+  .connect(process.env.DATABASE)
+  .then(() => console.log('Database has been connected successfully'));
```
Comment (belyas) [inline server.js:19]: It's better if you check if this is on non-production environment to console log this text

**Review by belyas — CHANGES_REQUESTED** [review]

**Comment by belyas** [comment]
@idurar any idea why yarn locks aren't already on the repo? I'm new to this project and I can see that this PR introduces `yarn.lock` files in both the root dir and frontend dir

File: frontend/src/style/partials/header.less (hunk 4 above)
Comment (zeeshanibnali) [reply frontend/src/style/partials/header.less:18]: oops, my bad. This is indeed the correct way, I don't know how but I forgot... Thanks brother.

File: server.js (hunk 5 above)
Comment (zeeshanibnali) [reply server.js:19]: Could you please elaborate what are the problems if this text is logged in a production environment, I am just asking out of curiosity. = )

Comment (belyas) [reply server.js:19]: All console logs/errors must not be logged on production as a general rule, first of all, it's noise on the devtools console especially such logs, there are some logs are may expose some platform content, typically console logs/infos are for development only, console warnings/errors must be used with a logger/monitoring service instead and keep devtools console only for uncaught messages

Comment (zeeshanibnali) [reply server.js:19]: You are right!

**Comment by polymahh** [comment]
this issue was fixed in this [PR](https://github.com/idurar/idurar-erp-crm/pull/235)

**Pull request closed without merging**